ios/FluentUI/AvatarGroup/AvatarGroup.swift

```diff
@@ -114,14 +114,24 @@ public struct AvatarGroup: View, ConfigurableTokenizedControl {
         let hasOverflow: Bool = overflowCount > 0
         let isStackStyle = tokens.style == .stack
 
+        let overflowAvatar = createOverflow(count: overflowCount)
+
         let interspace: CGFloat = tokens.interspace
-        let ringOuterGap: CGFloat = tokens.ringOuterGap
-        let ringGapOffset: CGFloat = ringOuterGap * 2
-        let ringOffset: CGFloat = tokens.ringThickness + tokens.ringInnerGap + tokens.ringOuterGap
 
-        let overflowAvatar = createOverflow(count: overflowCount)
-        let imageSize: CGFloat = overflowAvatar.contentSize
-        let groupHeight: CGFloat = imageSize + (ringOffset * 2)
+        let groupHeight: CGFloat = {
+            let avatarMaxHeight: CGFloat
+            if let avatar = avatarViews.first {
+                let avatarTokens = avatar.tokens
+                avatarMaxHeight = avatarTokens.avatarSize + 2 * (avatarTokens.ringThickness + avatarTokens.ringInnerGap + avatarTokens.ringOuterGap)
+            } else {
+                avatarMaxHeight = 0
+            }
+
+            let overflowTokens = overflowAvatar.tokens
+            let overflowMaxHeight = overflowTokens.avatarSize + 2 * (overflowTokens.ringThickness + overflowTokens.ringInnerGap + overflowTokens.ringOuterGap)
+
+            return max(avatarMaxHeight, overflowMaxHeight)
```

Comment:
If **hasOverflow** is false, we shouldn't consider it in this calculation, right?

Reply:
I think we still want to? In case the overflow shows up later, we wouldn't want the size of the whole control to change. Also, if there's not overflow and no avatars, then the size would be zero, and I don't think we want that, either.

Reply:
If the overflow shows up later this will be recalculated as it's part of the body property of the view.
The scenario of no avatars (regular or overflow) is unlikely, but I agree it'd be zero. I don't think we have a clear way of defining a constant height in those scenarios. Because the tokens could be overridden too. So we don't know whether the overflow could be larger or smaller than the other avatars.

Reply:
Nevermind this comment... 
The overflow avatar is the default case here as we talked offline (in case there are no other avatars).
I think we can stick with that now!
Thanks!